src/proto/grpc/gcp/handshaker.proto

```diff
@@ -109,6 +109,9 @@ message StartClientHandshakeReq {
   // (Optional) Maximum frame size supported by the client.
   uint32 max_frame_size = 10;
 
+  // (Optional) Internal use only.
+  string access_token = 11;
```

Comment:
Note that the authoritative copy of this proto is in https://github.com/grpc/grpc-proto/blob/master/grpc/gcp/handshaker.proto.  The copy we have here is just lingering tech debt that we ultimately want to get rid of.  So please make sure that any changes that happen here are also reflected in the authoritative copy.

Reply:
Yes. The field has already been introduced there. Actually there is yet another copy of the proto in the `src/coretsi/alts/handshaker/proto`. I think it's dead code but will defer to a follow-up PR to clean that up.